reorganize.py

```diff
@@ -0,0 +1,169 @@
+        'total_presym_infection',
+        'total_sym_infection',
+        'n_remaining_popsize_3499',
+        'n_no_outbreak',
+        'n_no_infected_by_seed',
+        'n_seed_show_no_symptom',
+        'n_no_first_infection',
+        'n_first_infection_on_day_1',
+        'n_first_infection_on_day_2',
+        'n_first_symptom',
+        'n_first_symptom_on_day_1',
+        'n_second_symptom',
+        'n_second_symptom_on_day_1',
+        'n_third_symptom',
+        'n_third_symptom_on_day_1',
+        ]
+
+        for i in del_list:
+            del pq_dict[i]
+        print('Extraneous Deleted')
+    def cellFill(key): #fills in cells given dict key; exception case checking for n_outbreak_duration because it has a list of 1 element
+        keysplit = key.split('_')
+        row = int(float(keysplit.pop()))
+        column = '_'.join(keysplit)
+        if column in avg_header_list:
+            avg_value = pq_dict[key].strip('\n').split(',')
+            pq_df.at[row,column] = avg_value[0]
+            if len(avg_value) > 1: 
+                pq_df.at[row,column + '_1'] = avg_value[1]
+        if column == 'n_recovered':
+            l_value = (pq_dict[key].strip('\n').split(', '))
+            for i in l_value:
+                rec_value = i.split(':')
+                pq_df.at[row,'n_recovered_'+ rec_value[0]] = rec_value[1]
+        if column == 'n_infected':
+            l_value = (pq_dict[key].strip('\n').split(', '))
+            for i in l_value:
+                inf_value = i.split(':')
+                pq_df.at[row,'n_infected_'+ inf_value[0]] = inf_value[1]
+        if column == 'n_active':
+            l_value = (pq_dict[key].strip('\n').split(', '))
+            for i in l_value:
+                act_value = i.split(':')
+                pq_df.at[row,'n_active_'+ act_value[0]] = act_value[1]
+        if column == 'n_popsize':
+            l_value = (pq_dict[key].strip('\n').split(', '))
+            for i in l_value:
+                pop_value = i.split(':')
+                pq_df.at[row,'n_popsize_'+ pop_value[0]] = pop_value[1]
+        if column == 'incidence_rate':
+            l_value = (pq_dict[key].strip('\n').split(', '))
+            for i in l_value:
+                inc_value = i.split(':')
+                pq_df.at[row,'incidence_rate_'+ inc_value[0]] = inc_value[1]
+        if column == 'seroprevalence':
+            l_value = (pq_dict[key].strip('\n').split(', '))
+            for i in l_value:
+                ser_value = i.split(':')
+                pq_df.at[row,'seroprevalence_'+ ser_value[0]] = ser_value[1]
+
+    #main
+    
+    pq_dict = {}         
+    createDictionary(file)
+
+    #creates index
+    l = []
+    seperator = ''
+    for x in pq_dict.keys():
+        result = (seperator.join(re.findall(r'[1-9]', x)))
+        if result.isdigit():
+            l.append(int(result))
+
+    dayslist = list(range(max(l) + 1)) 
+    print('Max Duration Found: '+ str(max(l)))

```

Comment:
Yes, that is the idea. First find the `dayslist`, then find all rows with `dayslist` , then decide on column names from the type of values.